webapp/migrations/versions/292882794509_add_timezone_to_last_modified.py

```diff
@@ -17,7 +18,8 @@
 
 
 def upgrade():
-    op.alter_column('user', 'last_modified', type_=sa.TIMESTAMP(timezone=True))
+    if current_app.config['ENV'] in ('staging', 'production'):
+        op.alter_column('user', 'last_modified', type_=sa.TIMESTAMP(timezone=True))
```

Comment:
Are you maybe reading the condition the wrong way around? It does nothing in the environments that use SQLite, that's why there's no error. :)
(BTW: I haven't found an easy way to get the dialect name, that's why the if branches on `ENV`. Would of course be nice to say `if not something.dialect.name == "sqlite":`, but I don't know how.)

Reply:
Frede suggested a very pragmatic approach: just look in the SQL connection string!

Reply:
Yes, I read it the wrong way. Nevertheless, I am still a little confused. This would then skip this for SQLite. So, we do not get an error but this does not fix the migration problem, does it? I thought we would need to change the command for that (which is basically what I have done locally but I could not figure out how to do for all of us without breaking things fast enough).

I have referred to [this blogpost](https://blog.miguelgrinberg.com/post/fixing-alter-table-errors-with-flask-migrate-and-sqlite) quite a lot. But I might be mistaken completely about the problem you are trying to solve.

Reply:
You're completely right, of course - this just makes the error go away. However, we only added the timezone because we had this peculiar situation in production where SysEleven set the server time zone of the database server to CET while the clients run on UTC. This mismatch caused subtle problems. SQLite always runs where the tests run, so this should never be an issue. What's more: [SQLite doesn't even support timestamp types](https://www.sqlite.org/datatype3.html), let alone timezones, so getting this exactly right would be quite a project… So for now, I'd take the pragmatic route of just skipping those migrations.